explorer/scan/local_scanner.go

```diff
@@ -201,7 +201,8 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstreamConf
 	if isatty.IsTerminal(os.Stdout.Fd()) && !strings.EqualFold(logger.GetLevel(), "debug") && !strings.EqualFold(logger.GetLevel(), "trace") {
 		multiprogress, err = progress.NewMultiProgressBars(progressBarElements, orderedKeys)
 		if err != nil {
-			return nil, false, errors.Wrap(err, "failed to create progress bars")
+			log.Error().Err(err).Msg("failed to create progress bars; using noop progress bar")
+			multiprogress = progress.NoopMultiProgressBars{}
```

Comment:
We should figure out why this error happened

Reply:
This error happened because of the problem with the asset.
I pass two lists to the progress bars. The first list contains the assets. The second list contains the keys in which order the progress bars should be shown. A key is a platformID: https://github.com/mondoohq/cnquery/blob/main/explorer/scan/local_scanner.go#L195

Reply:
@vjeffrey Could you please add the command, which triggered the error? I'd give it a try to reproduce it.

Reply:
This is another way to prevent the error: https://github.com/mondoohq/cnquery/pull/860/files
this way, we would at least get the progress bars for valid assets.

But it also boils down to the question below, why we have the nil asset.

Reply:
i think i know why, there was a nil ecs cluster that i had picked up, i'll work on a fix for that

Reply:
i'm pushing a fix to https://github.com/mondoohq/cnquery/pull/857